packages/math-new/src/index.ts

```diff
@@ -41,19 +41,19 @@ export class AMM {
     switch (amm.poolType) {
       case "MetaStable":
         this.math = new ExtendedMetaStableMath(
-          amm.poolParams as IMetaStableMath,
+          amm.poolParams as IMetaStableMath
         );
       case "MetaStable_2":
         this.math = new ExtendedMetaStableMath_2(
-          amm.poolParams as IMetaStableMath_2,
+          amm.poolParams as IMetaStableMath_2
         );
     }
   }
 
   exactTokenInForTokenOut(
     amountIn: number,
     tokenIn: string,
-    tokenOut: string,
+    tokenOut: string
```

Comment:
think this is probably a prettier change 🤔 have you run prettier in this PR after merging back from the target branch?